include/linker/common-rom.ld

```diff
@@ -67,13 +67,22 @@
 		__net_l2_end = .;
 	} GROUP_LINK_IN(ROMABLE_REGION)
 
+#if defined(CONFIG_NET_SOCKETS)
+	SECTION_PROLOGUE(net_socket_register, (OPTIONAL),)
+	{
+		__net_socket_register_start = .;
+		*(".net_socket_register")
+		KEEP(*(SORT_BY_NAME(".net_socket_register*")))
+		__net_socket_register_end = .;
+	} GROUP_LINK_IN(ROMABLE_REGION)
+#endif
+
 	SECTION_DATA_PROLOGUE(_bt_services_area,,SUBALIGN(4))
 	{
 		_bt_services_start = .;
 		KEEP(*(SORT_BY_NAME("._bt_services.static.*")))
 		_bt_services_end = .;
 	} GROUP_LINK_IN(ROMABLE_REGION)
-
```

Comment:
Why this line was removed?

Reply:
Good catch. I missed this one when solving merge conflict. Will fix.